### ISSUE TITLE
[Breaking Change] string WhereX methods should not use the LOWER method by default

### ISSUE THREAD

**Issue by ahmad-moussawi** [sqlkata/querybuilder#610]
Currently SqlKata will try to apply a case insensitive operation, for WhereString operations, 
for example this query
```
var query = new Query("Car").WhereLike("Name", "AM");
```

will get translated to 

```
SELECT * FROM [Car] WHERE LOWER([Name]) like 'am'
```

What requested is to keep the default behavior of the database, and only apply the transformation if explicitly needed

**Comment by FrancescoBonizzi** [comment]
I agree with @ahmad-moussawi. Also, calling `LOWER` on every parameter has a significant performance overhead.
